apps/src/templates/projects/projectsRedux.js

```diff
@@ -302,6 +327,57 @@ export function updateProjectName(projectId, updatedName) {
   return {type: UPDATE_PROJECT_NAME, projectId, updatedName};
 }
 
+export function saveSuccess(projectId) {
```

Comment:
style nit: we tend to organize reducers so that the action types (i.e., SAVE_SUCCESS/SAVE_FAILURE) and the reducers (i.e., saveSuccess/saveFailure) are lumped together near the top of the file. a good example is [manageStudentsRedux](https://github.com/code-dot-org/code-dot-org/blob/903884e3e9b1413504373294f1a76586691534f7/apps/src/templates/manageStudents/manageStudentsRedux.js#L119-L167), since it has a ton of actions